static/src/stylesheets/module/business/_stocks.scss

```diff
@@ -0,0 +1,170 @@
+$stocks-icon-width: 14px;
+$stocks-icon-height: 10px;
+$closed-colour: #767676;
```

Comment:
`#767676` === `guss-colour(neutral-2, $pasteup-palette)` so no need to store in `var`